extensions/interactions/LogicProof/LogicProof.html

```diff
@@ -18,9 +18,10 @@
         <textarea ng-trim="false" rows="10"></textarea>
       </div>
     </div>
-    <button class="btn btn-default" ng-click="submitProof()">Submit</button>
+    <!-- The onclick for proof submission is added in the javascript -->
+    <button class="btn btn-default logic-proof-submit">Submit</button>
     <button class="btn btn-default" ng-click="showHelp()">Possible lines</button>
-    <p class="logic-proof-mistake-message"> <[proofError]>  </p>
+    <p class="logic-proof-mistake-message"> <[mistakeMessage]>  </p>
```

Comment:
This commit is just to restore lost functionality; it does not address improvements to the UI.

We can certainly consider changes in future, though I'm not keen on the codemirror example because it does not fit the paradigm of there being a single error at a time, and users may not realise that the tooltips exist.